mail/mail-server/Portfile

```diff
@@ -47,9 +47,19 @@ depends_lib-append      port:apache-solr8 \
 
 depends_run-append      port:clamav-server
 
-variant initialize \
-    description {Initialize all configuration files. Existing
-        configurations files are not overwritten by default.} {}
+variant initializealways \
```

Comment:
Other variants with multiple words use an underscore. So change this to `initialize_always`

Reply:
Thanks, done.

Reply:
And also with https://github.com/macports/macports-ports/pull/4978 and https://github.com/macports/macports-ports/pull/5643.